lib/date.js

```diff
@@ -151,6 +182,32 @@ internals.Date.prototype.iso = function () {
     return obj;
 };
 
+internals.Date.prototype.timestamp = function (unit) {
+
+    const obj = this.clone();
+
+    // set unit if passed (overwrites _unit)
+    if (unit) {
+        obj._unit = unit;
+    }
+
+    // validate unit is allowed
+    if (obj._unit) {
+        internals.isValidUnit(obj._unit);
+    }
+
+    obj._flags.timestamp = true;
```

Comment:
Stick to the format field, I don't like the fact that flag can conflict each other and introduce priority.